docker/local.conf

```diff
@@ -0,0 +1,51 @@
+BUILDCFG_VARS = "VARIANT_NAME VARIANT_ID BB_VERSION BUILD_SYS NATIVELSBSTRING TARGET_SYS MACHINE DISTRO DISTRO_VERSION TUNE_FEATURES TARGET_FPU"
+DISTRO = "printnanny"
+IMAGE_FSTYPES = "tar.gz ext4 ext4.gz ext4 wic wic.gz wic.bmap"
+LICENSE_FLAGS_ACCEPTED += "\
+    commercial_gstreamer1.0-libav \
+    commercial_gstreamer1.0-plugins-ugly \
+    commercial_ffmpeg \
+    commercial_mpeg2dec \
+    synaptics-killswitch \
+"

```

Comment:
I do wonder if these should be dropped in what's committed to the repo, since presumably users would need to accept these licenses on their own behalf?

Reply:
These can probably be moved into PrintNanny's distro conf file `printnanny.conf` - but this is working, for now, let's just roll with it and we can circle back to licensing on Lawyer Day.